cg/apps/cgstats/stats.py

```diff
@@ -145,6 +145,8 @@ def fastqs(self, flowcell: str, sample_obj: Sample) -> Iterator[Path]:
         """Fetch FASTQ files for a sample."""
         base_pattern = "*{}/Unaligned*/Project_*/Sample_{}/*.fastq.gz"
         alt_pattern = "*{}/Unaligned*/Project_*/Sample_{}_*/*.fastq.gz"
-        for fastq_pattern in (base_pattern, alt_pattern):
+        bcl_convert_pattern = "*{}/Unaligned*/*/Sample_{}_*/*.fastq.gz"

```

Comment:
We never added the pattern for novaseqx pattern in the old flow. I guess that is fine if we ever manage to delete the old flow 😅

Reply:
So what this does is introduces another folder structure type in the list of valid structures and if it matches with it assigns a bcl_converter value?

Reply:
Oh, now I understand. So the function applies all the patterns and return any file path which match any of the patterns